src/common/models/Projection.ts

```diff
@@ -309,6 +311,25 @@ export class Projection {
     return false;
   }
 
+  get twoWeekPercentChangeInCases() {
+    return this.getTwoWeekPercentChange(this.smoothedDailyCases);
+  }
+
+  get twoWeekPercentChangeInDeaths() {
+    return this.getTwoWeekPercentChange(this.smoothedDailyDeaths);
+  }
+
+  private getTwoWeekPercentChange(series: any[]): number | null {
+    const lastIndex = indexOfLastValue(series) ?? series.length;
+    const lastTwoWeeks = series.slice(lastIndex - 14, lastIndex + 1);
```

Comment:
Should we use the dates to calculate the value 2 weeks ago instead? For the U.S. series this is probably okay, but for counties with sparse data the date corresponding to 14 days ago could be months ago (for data reported weekly).

We can punt if this is not really a problem, but maybe add a note so we know that this calculation is assuming that we have one data point per day.

Reply:
We actually have a bunch of code that assumes that these timeseries don't have gaps and there is a (possibly null) data point for every day.

This is guaranteed to work because of [`getAlignedTimeseriesAndDates()`](https://github.com/covid-projections/covid-projections/blob/94bd2ae85096794f15ecd77a63ea8cba59b9420d/src/common/models/Projection.ts#L494)